task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -111,7 +111,7 @@ spec:
         . /utils.sh # gives us the make_result_json helper used below.
 
         # Generate TEST_OUTPUT
-        TEST_OUTPUT=$(make_result_json -r "SKIPPED" -t "${NOTE}")
+        TEST_OUTPUT=$(make_result_json -r "SUCCESS" -t "${NOTE}")
```

Comment:
Maybe a comment:

```suggestion
        # Don't use "SKIPPED" because it produces unwanted Conforma violations
        TEST_OUTPUT=$(make_result_json -r "SUCCESS" -t "${NOTE}")
```